bool find
---
bool success
geometry_msgs/Pose currentPos
geometry_msgs/Pose nextPos